pkg/v1/tkg/client/init.go

```diff
@@ -342,7 +342,8 @@ func (c *TkgClient) InitRegion(options *InitRegionOptions) error { //nolint:funl
 		log.Warningf("Warning: Management cluster is created successfully, but some packages are failing. %v", err)
 	}
 
-	log.Infof("Context set for management cluster %s as '%s'.", options.ClusterName, kubeContext)
+	log.Infof("Context set for management cluster %s as '%s' in tkg's kubeconfig.", options.ClusterName, kubeContext)
```

Comment:
```suggestion
	log.Infof("Context set for management cluster %s as '%s' in '%s' kubeconfig.", options.ClusterName, kubeContext, tkgKubeconfigFile)
```

I am not super comfortable with using `tkg's` kubeconfig as not many user will know what that mean.

Reply:
Or I would prefer to completely remove this log line to avoid confusion if we are adding the below log line.

Reply:
Yeah fair point. I'll just remove this log line